### PR TITLE
MS-195: Psychologists first view after login

### PR DESCRIPTION
**Pull request by BareaMartin**

### CODE REVIEW

**Review by AndresCespedes23 — APPROVED** [review]

File: src/Components/Psychologist/FirstView/first-view.module.css [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+}
+.dates {
+  padding-left: 1vw;
+}
+.userName {
+  padding-left: 1vw;
+  font: bold 20px Century Gothic;
+  width: 15vw;
+}
+.assignBtn {
```
Comment (julianv97) [inline src/Components/Psychologist/FirstView/first-view.module.css:77]: missing cursor: pointer for the buttons

File: src/Components/Psychologist/FirstView/index.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        <button className={styles.backBtn}>BACK</button>
+        <div className={styles.content}>
+          <h3 className={styles.title}>
+            <span className={styles.bold}>Unassigned users:</span>
+          </h3>
+          <table>
+            <tbody>
+              {postulants.map((postulant) => {
+                if (postulant.status === 'PENDING INTERVIEW') {
+                  return [
```
Comment (julianv97) [inline src/Components/Psychologist/FirstView/index.js:28]: this returns should not be an array, you have tu use ()

File: src/Components/Psychologist/FirstView/index.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        </div>
+        <div className={styles.content}>
+          <h3 className={styles.title}>
+            <span className={styles.bold}>Upcoming Interviews:</span>
+          </h3>
+          <table>
+            <tbody>
+              {sessions.map((session) => {
+                if (session.status === 'PENDING') {
+                  return [
```
Comment (julianv97) [inline src/Components/Psychologist/FirstView/index.js:49]: same return here

**Review by julianv97 — APPROVED** [review]

**Review by julianv97 — CHANGES_REQUESTED** [review]
Le erré al botón de pedir cambios en la anterior

**Review by julianv97 — APPROVED** [review]

**Pull request merged**